pkg/bundle/sync.go

```diff
@@ -23,6 +23,7 @@ import (
 	"encoding/hex"
 	"errors"
 	"fmt"
+	"k8s.io/apimachinery/pkg/labels"
```

Comment:
This import should be moved to the preceding block. This block is for stdlib imports.

Reply:
Done!